client/checkout/classic/upe.js

```diff
@@ -116,11 +116,9 @@ jQuery( function ( $ ) {
 	const hiddenBillingFields = {
 		name: 'never',
```

Comment:
Just want to confirm, all these other fields will always be required during checkout with UPE right? I am not sure if the customer would want to hide one of these remaining fields as well using customizer, else it will throw a similar error.

Reply:
I was trying to some docs for this options object, I found this page https://stripe.com/docs/js/elements_object/create_element?type=card

But wasn't about to find docs for the `payment` element type that we use here https://github.com/Automattic/woocommerce-payments/blob/c2dd9bbaab9d337e933e1a1c168740d1cc46965b/client/checkout/classic/upe.js#L302

![image](https://user-images.githubusercontent.com/1527164/132167554-59c222ba-ac57-4a2e-965e-dcf10a2a1a3a.png)

Is it because this feature is in private beta? Any ideas?

Reply:
We backtraced the PR which was responsible for adding this code. We found that there is no official documentation for this feature as this is in beta and stripe proved information about this may be via google docs. More details here: https://github.com/Automattic/woocommerce-payments/issues/1981